modules/ztp-validating-the-generation-of-policy-crs.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-validating-the-generation-of-policy-crs_{context}"]
+= Validating the generation of policy CRs
+
+ArgoCD generates the policy custom resources (CRs) in the same namespace as the PolicyGenTemplate from which they were created. The same troubleshooting flow applies to all policy CRs generated from PolicyGenTemplates regardless of whether they are common, group, or site based.
```

Comment:
s/PolicyGenTemplate/`PolicyGenTemplate` (throughout) 

The check status step should be moved under procedure.

"To check the status of the policy CRs, enter the following command:"

Reply:
Changed.
Did not move the check status step.